jobs/async-upload/job/config.py

```diff
@@ -101,6 +107,9 @@ def _parser() -> cap.ArgumentParser:
     p.add_argument("--registry-custom-ca-envvar", default=None)
     p.add_argument("--registry-log-level", default=logging.WARNING)
 
+    # --- ConfigMap metadata ---
+    p.add_argument("--metadata-configmap-path", default=None, help="Path to mounted ConfigMap with model metadata")
```

Comment:
this is fine as is for now, but wondering (in followup PRs) if best to provide a default as `DEFAULT_METADATA_CONFIGMAP_PATH`, also add it in README?